GNUmakefile

```diff
@@ -173,21 +175,15 @@ bootstrap:
 kubegen2: deepcopy2 clientset2 lister2 informer2
 
 # code generation for custom resources
-kubegen: deepcopy clientset lister informer kubegen2
+kubegen1: 
+	./buildscripts/code-gen.sh ${GENS} ${OUTPUT_PKG} ${APIS_PKG} ${GROUPS_WITH_VERSIONS} --go-header-file ${BOILERPLATE_TEXT_PATH} 
```

Comment:
Can we have a new make target that will download this code-gen.sh file and override the existing. This will ensure we have a way to download the latest file.

Reply:
As discussed, we have to put only the base file information.